sourcecode/hub/resources/views/content/standalone-view.blade.php

```diff
@@ -1,71 +1,64 @@
-                    <button type="button" class="btn btn-primary mx-2" data-bs-toggle="modal" data-bs-target="#reportSuccessModal" aria-label="Report Content"> {{ trans('messages.report-content')}}</button>
-                    <button type="button" class="btn btn-secondary mx-2" data-bs-dismiss="modal" aria-label="Cancel"> {{ trans('messages.cancel')}}</button>
+                    <button type="button" class="btn btn-primary mx-2" data-bs-toggle="modal" data-bs-target="#reportSuccessModal" aria-label="{{ trans('messages.report-content')}}"> {{ trans('messages.report-content')}}</button>
+                    <button type="button" class="btn btn-secondary mx-2" data-bs-dismiss="modal" aria-label="{{ trans('messages.cancel')}}"> {{ trans('messages.cancel')}}</button>
                 </div>
             </div>
         </div>
     </div>
-
     <!-- Report Content Success Modal -->
-    <div class="modal fade" id="reportSuccessModal" tabindex="-1" aria-labelledby="reportSuccessModalLabel" aria-hidden="true" data-bs-backdrop="static">
+    <div class="modal fade" id="reportSuccessModal" tabindex="-1" aria-labelledby="{{ trans('messages.report-success-modal-label')}}" aria-hidden="true" data-bs-backdrop="static">
         <div class="modal-dialog modal-dialog-centered">
             <div class="modal-content">
                 <div class="modal-header border-0 text-center">
-                    <i class="bi bi-check-circle fs-1 mx-auto" aria-label="Check Mark Icon"></i>
+                    <x-icon name="check-circle" id="reportSuccessModalLabel" class="bi fs-1 mx-auto" aria-label="{{ trans('messages.check-mark-icon')}}" />
                 </div>
                 <div class="modal-body text-center">
-                    <h3 class="font-weight-bold mb-3" aria-label="Reported Successfully">{{ trans('messages.reported-successfully')}}</h3>
+                    <h3 class="font-weight-bold mb-3" aria-label="{{ trans('messages.reported-successfully')}}">{{ trans('messages.reported-successfully')}}</h3>
                     <p>{{ trans('messages.we-will-look-into-this-shortly')}}</p>
                     <p>{{ trans('messages.thanks-for-reporting')}}</p>
                 </div>
                 <div class="modal-footer border-0 d-flex justify-content-center align-items-center py-4">
-                    <button type="button" class="btn btn-primary mx-2" data-bs-dismiss="modal" aria-label="Go Back">{{ trans('messages.go-back')}}</button>
-                    <button type="button" class="btn btn-secondary mx-2" aria-label="Go to Edlib">{{ trans('messages.go-to-edlib')}}</button>
+                    <button type="button" class="btn btn-primary mx-2" data-bs-dismiss="modal" aria-label="{{ trans('messages.go-back')}}">{{ trans('messages.go-back')}}</button>
+                    <button type="button" class="btn btn-secondary mx-2" aria-label="{{ trans('messages.go-to-edlib')}}">{{ trans('messages.go-to-edlib')}}</button>
```

Comment:
The content of the element is a readable text, no need to repeat it using `aria-label`